charts/stable/autobrr/values.yaml

```
@@ -0,0 +1,87 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: ghcr.io/autobrr/autobrr
+  # @default -- chart.appVersion
+  tag: v0.23.0
```

```suggestion
  tag:
```

By default appVersion from the chart.yaml is used